WooCommerce/Classes/POS/Presentation/PointOfSaleDashboardView.swift

```diff
@@ -59,6 +59,37 @@ struct PointOfSaleDashboardView: View {
                 }
             }
         })
+        .posModal(isPresented: $viewModel.showExitPOSModal, fixedWidth: false, fixedHeight: false, content: {
+            VStack(spacing: 0 ) {
+                HStack {
+                    Spacer()
+                    Button {
+                        viewModel.showExitPOSModal = false
+                    } label: {
+                        Image(PointOfSaleAssets.xClose.imageName)
+                            .frame(width: Constants.closeIconSize, height: Constants.closeIconSize)
+                            .foregroundColor(Color.posTertiaryTexti3)
+                    }
+                }
+                Text("Exit Point of Sale mode?")
+                    .font(.posModalTitle)
+                    .padding(.bottom)
+                Text("Any orders in progress will be lost.")
+                    .font(.posModalBody)
+                    .padding(.bottom)
+                    .padding(.bottom)
+                    .padding(.bottom)
```

Comment:
Is this ok? I was trying to avoid writing explicit numbers for padding but wanted to make it be like on Figma designs.

Reply:
I think it's fine to use an explicit number for padding.

Reply:
You can define it in one of the shared Layout structs, or the `Constants` enum for this view